Modules/Core/Common/include/itkGaussianOperator.h

```diff
@@ -142,6 +142,19 @@ class ITK_TEMPLATE_EXPORT GaussianOperator : public NeighborhoodOperator<TPixel,
     os << indent << "MaximumKernelWidth: " << m_MaximumKernelWidth << std::endl;
   }
 
+  /** Get the value of the debug flag.  */
+  bool
+  GetDebug()
+  {
+    return m_Debug;
+  }
+  /** Set the value of the debug flag. A non-zero value turns debugging on. */
+  void
+  SetDebug(bool debugFlag)
```

Comment:
I didn't check similar functions in ITK, so maybe this is a stupid question.  Would it be appropriate to have `debugFlag` default to `true`?

Reply:
Good question, `debugFlag` defaults to `false` in `itk::Object` and should be the same here. We want developers to be able to access extra debug info when something is going wrong, but the original motivation in #3423 was to make `itk::GaussianOperator` less verbose because the "warning" was simply informing the user that the kernel was behaving according to user specifications.

Reply:
Definitely `m_Debug` should default to `false`.  My question was about the `debugFlag` argument to `SetDebug`, so that `this->SetDebug(true)` and `this->SetDebug()` do the same thing.

Reply:
See above -- I can't find other examples of `this->SetDebug()` in ITK and I would rather avoid introducing it here as I don't think the default parameter is intuitive.

Reply:
I have added `DebugOn` and `DebugOff` as alternatives requiring no parameters. These mimic the interface in `itk::Object`.